### PR TITLE
Added Node name and paths to errors

### LINKED ISSUE #20: add node name in beehave error message
**Is your feature request related to a problem? Please describe.**
The error `Beehave error: Root should have one child` (beehave_root.gd:16) comes without a hit or something similar to what node is missing the child, making it harder to identify the origin of the error

**Describe the solution you'd like**
`Beehave error: Root should have one child in node " + str(name)`

**Describe alternatives you've considered**
Some way to link to the node instead of just printing the name, but I don't know how one would do that or if that is even possible.

### PR DESCRIPTION
**Pull request by Nemesis-AS**
## Description

This PR adds NodePaths to the error messages to make it easier to find the node throwing the error

## Addressed issues

- Fixes #20 

## Screenshots

BehaviourTreeScene tscn - Godot Tutorial_ Behaviour Trees - Godot Engine 20-08-2022 13_24_16

## Checklist

- [x] changes performed compatible with Godot 3.x
- [ ] changes performed compatible with Godot 4.x (raised in a separate PR against the `2.x` branch in case the change is not compatible with Godot 3.x like specific syntax)
- [ ] Unit tests (Godot 4 only after https://github.com/bitbrain/beehave/issues/2 is done)

### CODE REVIEW

**Comment by Nemesis-AS** [comment]
On second thought, I think the NodePaths should be moved to the end so the error message doesn't gets truncated. Like so 
`Beehave Error: Message Body (NodePath: /root/node/path)`

File: addons/beehave/nodes/decorators/decorator.gd [hunk 1]
```diff
@@ -5,4 +5,4 @@ class_name Decorator, "../../icons/category_decorator.svg"
 
 func _ready():
 	if self.get_child_count() != 1:
-		push_error("Beehave Error: Decorator %s should have only one child" % self.name)
+		push_error("Beehave Error: Decorator %s (%s) should have only one child" % [self.name, self.get_path])
```
Comment (bitbrain) [inline addons/beehave/nodes/decorators/decorator.gd]: you are also missing brackets here

Comment (Nemesis-AS) [reply addons/beehave/nodes/decorators/decorator.gd]: I'm sorry but which brackets are you referring to 😅

Comment (bitbrain) [reply addons/beehave/nodes/decorators/decorator.gd]: `get_path()` instead of `get_path`

Comment (Nemesis-AS) [reply addons/beehave/nodes/decorators/decorator.gd]: Oh those ones, thanks for pointing that out

**Comment by bitbrain** [comment]
@Nemesis-AS since there aren't any tests existing yet, did you test these changes with the latest Godot 3 version?

**Comment by Nemesis-AS** [comment]
> @Nemesis-AS since there aren't any tests existing yet, did you test these changes with the latest Godot 3 version?

I did test the errors in the starter project from your tutorial, but let me just spin up a new project quickly and see if it works

**Comment by Nemesis-AS** [comment]
Main tscn - Beehave Test - Godot Engine 20-08-2022 21_12_08
Works on a new Empty Project with Godot 3.5

**Review by bitbrain — APPROVED** [review]

**Comment by bitbrain** [comment]
Good job! I'll cherry-pick this onto the `godot-4.x` branch for you!

**Pull request merged**